app/src/main/java/com/sunasterisk/music_72/data/source/remote/TrackRemoteDataSource.kt

```diff
@@ -0,0 +1,27 @@
+package com.sunasterisk.music_72.data.source.remote
+
+import com.sunasterisk.music_72.data.model.Track
+import com.sunasterisk.music_72.data.source.TrackDataSource
+import com.sunasterisk.music_72.data.source.remote.connection.RetrofitClient
+import io.reactivex.Observable
+
+class TrackRemoteDataSource(retrofitClient: RetrofitClient) : TrackDataSource.remote {
+
+    private val trackService = retrofitClient.getTrackService()
+
+    override fun getAllTrack(): Observable<List<Track>>? =
```

Comment:
tương tự cho các fun ở dưới
```suggestion
    override fun getAllTrack(): Observable<MutableList<Track>>? =
```